apps/backoffice/src/components/cta-manager/service-cta-manager.tsx

```diff
@@ -0,0 +1,154 @@
+import { Banner } from "@/components/banner";
+import { FormStepSectionWrapper } from "@/components/forms";
+import {
+  SelectController,
+  TextFieldController,
+  UrlFieldController,
+} from "@/components/forms/controllers";
+import { Link, ReportProblemRounded } from "@mui/icons-material";
+import { Box, Divider, Grid } from "@mui/material";
+import { useTranslation } from "next-i18next";
+import React from "react";
+import { useFormContext } from "react-hook-form";
+
+import ButtonAddRemove from "../buttons/button-add-remove";
+
+export const ServiceCtaManager: React.FC = () => {
+  const { t } = useTranslation();
+  const { clearErrors, setValue, watch } = useFormContext();
+
+  const hasCta2UrlPrefix = watch("metadata.cta.cta_2.urlPrefix");
+  const initialStateButtonRemove = hasCta2UrlPrefix !== "" ? true : false;
+  const selectItems = [
+    {
+      label: t("forms.service.extraConfig.cta.form.externalLink"),
+      value: "iohandledlink://",
```

Comment:
this label (iohandledlink://) as other label are heavily duplicated within the code, why don't you declare constants?